### DISCUSSION TITLE
Mail Scanner - Can new tickets be created based on only unread emails (new emails)?

### DISCUSSION THREAD

**Discussion by afatac** [YetiForceCompany/YetiForceCRM#16945] [How to]
I try configuring the Mail Scanner today. The targeted test email account already has some emails within. So after the scan is activated, new tickets are created based on these existing emails.

This is not ideal. The actual email account that we are going to use already has hundreds of email messages. We do not want hundreds of new tickets being created.

Is there a way to configure so that new tickers are created only based on unread emails (new emails)?

**Comment by rskrzypczak** [comment]
There is no such option from the UI, but you can impose an email uid directly in the database from which to start scanning (vtiger_ossmailscanner_folders_uid)

**Reply by afatac** [reply]
Thanks for the suggestion. I found the table. But how do I know what uid to enter?

![image](https://user-images.githubusercontent.com/26587885/223523664-d05d88e7-f2f5-445e-94fe-1b91cc9c9926.png)

**Reply by rskrzypczak** [reply]
Well, there are many ways to do this, but the simplest is to open the mail client and find it from the page/frame source of the given email.

**Reply by afatac** [reply]
Hmm... after some extensive searching and reading in the Internet, it seems that it is not so simple to retrieve the uid. Coding knowledge is required.

In the email header, we can find the message-id of which is a long string of alphanumerical number. I do not think that is uid of which it seems more like a running number.

**Reply by mariuszkrzaczkowski** [reply]
the internet lies, look at this
![image](https://user-images.githubusercontent.com/10198654/224241292-8ebc6eb6-02ac-42e6-8f6a-01d94675c9db.png)

**Reply by afatac** [reply]
Ha ha, yes, the Internet lies. Thank you!
